examples/alarm/alarm.py

```diff
@@ -26,58 +27,84 @@
 # Create an alarm with a SET transition
 create_request = CreateOrUpdateAlarmRequest(
     alarm_id=alarm_id,
-    transition=CreateAlarmTransition(
-        transition_type=AlarmTransitionType.SET,
+    transition=SetAlarmTransition(
         occurred_at=datetime.now(),
-        severity_level=3,
+        severity_level=AlarmSeverityLevel.HIGH,
         condition="Temperature exceeded threshold",
-        message="Temperature sensor reading: 85°C",
     ),
 )
 # Returns instance_id - a server-generated unique identifier for this specific alarm occurrence
 id = client.create_or_update_alarm(create_request)
 
 # Get the alarm by its instance ID (the unique occurrence identifier)
-alarm = client.get_alarm(id)
+if id:
```

Comment:
See my comment about adding typing overloads. With those, you don't need this `if` because `create_or_update_alarm` never returns None when `ignore_conflict` is the default.